api/v1beta2/types.go

```diff
@@ -190,62 +190,62 @@ var (
 	ResourceTypeResourceGroup = ResourceType("resourceGroup")
 )
 
-// SecurityGroupRuleAction represents the actions for a Security Group Rule.
+// VPCSecurityGroupRuleAction represents the actions for a Security Group Rule.
 // +kubebuilder:validation:Enum=allow;deny
-type SecurityGroupRuleAction string
+type VPCSecurityGroupRuleAction string
 
 const (
 	// SecurityGroupRuleActionAllow defines that the Rule should allow traffic.
-	SecurityGroupRuleActionAllow SecurityGroupRuleAction = vpcv1.NetworkACLRuleActionAllowConst
+	SecurityGroupRuleActionAllow VPCSecurityGroupRuleAction = vpcv1.NetworkACLRuleActionAllowConst
```

Comment:
Do you wanna rename the variable names as well to keep them also in the same format?

Reply:
Yes renamed all of them, thanks!